CONTRIBUTING.md

```diff
@@ -1,12 +1,379 @@
-# Initial Contribution Guidelines - WIP
+# Primer
```

Comment:
Generally this is great and thank you.  As someone wrestling with trying to get the Orvibo motion detector, I realized right away that my replacement cluster didn't include my replacement code.

Generally I found sometimes the document described the what and not the why or how to use it.  I tried to give a few cases below.

Extra/later steps might include the a discussion on the timers required by some quirks and maybe a troubleshooting/debugging section.  (additional notes below).